backend/src/api/organization/organizationMerge.ts

```diff
@@ -8,7 +8,7 @@ export default async (req, res) => {
 
   const primaryOrgId = req.params.organizationId
   const secondaryOrgId = req.body.organizationToMerge
-  const segmentId = req.body.segments[0]
+  const segmentId = req.body.segments ? req.body.segments[0] : null
```

Comment:
the backup aggregates (like `activityCount`) won't be there if we don't send `segmentId` to here